c10/util/numa.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "c10/util/Logging.h"
+
+C10_DECLARE_bool(caffe2_cpu_numa_enabled);
+
+namespace c10 {
+
+C10_API bool IsNUMAEnabled();
```

Comment:
If we're going to move this to c10, let's document everything? :)

Reply:
Sure I'll add some brief documents, @ilia-cher could you write a more detailed one if there are more things to explain when this is landed?